docs/src/components/ComponentDoc/ComponentDocTag.tsx

```diff
@@ -1,27 +1,82 @@
 import * as React from 'react'
 import * as _ from 'lodash'
 import { Header, Message } from 'semantic-ui-react'
+const behaviorMenuItems = require('docs/src/behaviorMenu')
```

Comment:
what I am worrying about here is the following: currently we are using pretty much generic name for this accessibility concern, like just `behavior`. Next day we are planning to add state management functionality to our components, and there is a chance that this state management aspects will be called as `behaviors` as well, but those won't have anything to share with accessibility. In that case generic `behavior` name would be confusing, as it would have two different meanings.

Won't suggest to change anything now, but this is something that we should keep in mind

Reply:
FYI: @jurokapsiar

Reply:
yes, we can have that discussion and change it later. it might be accessibility behavior for example. I will put it on the agenda for one of the next meetings